core/commands/repo.go

```diff
@@ -259,7 +263,103 @@ daemons are running.
 	},
 	Type: MessageOutput{},
 	Marshalers: oldcmds.MarshalerMap{
-		oldcmds.Text: MessageTextMarshaler,
+		cmds.Text: MessageTextMarshaler,
+	},
+}
+
+var repoRmRootCmd = &cmds.Command{
```

Comment:
Doesn't this belong in the `ipfs files` command tree? Also, I'd just call it `ipfs files chroot` and allow the user to change the root arbitrarily.

However, I may just be missing some context here.

Reply:
We could even make it: `ipfs files chroot --save /oldroot /newroot` to allow saving the old root inside the new root. Actually, we might just want to do that by default to make this command less safer.

Reply:
> Doesn't this belong in the ipfs files command tree?

@Stebalien this is meant to be a low level tool to deal with the case when the root node some how gets deleted and is not available anywhere.  It is in `repo` because it a low level command. Like `repo fsck` or `repo verify`.

`ipfs files chroot` is an interesting idea, but more involved.

Reply:
We may want something like this for pin root too, as it can be useful in similar situations